relay-general/src/protocol/contexts/response.rs

```diff
@@ -0,0 +1,42 @@
+use crate::protocol::{Cookies, Headers};
+use crate::types::{Annotated, Object, Value};
+
+#[derive(Clone, Debug, Default, PartialEq, Empty, FromValue, IntoValue, ProcessValue)]
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct ResponseContext {
+    /// Type `response`.
+    #[metastructure(field = "type")]
+    pub ty: Annotated<String>,
```

Comment:
The type is injected by the `Contexts` wrapper, so this should not be added as a field here.